src/sentry/static/sentry/app/components/assigneeSelector.tsx

```diff
@@ -395,6 +511,7 @@ const MenuItemWrapper = styled('div')<{
 `;
 
 const InviteMemberLink = styled(Link)`
+  padding: 0 !important; /* Due to inconsistent styles when used as a child of <li> */
```

Comment:
![image](https://user-images.githubusercontent.com/1748388/66876159-9b6f2a80-ef65-11e9-89c2-2be852e97f06.png)

For bulk-assign, the `AssigneeSelector` is a child inside a dropdown and hence the first rule applied because it is more "specific".

Reply:
Booo less files strike again.

Reply:
You can do this I think:

```
  .dropdown-menu > li & {
    padding: 0; /* Due to inconsistent styles when used as a child of <li> */
  }
```

Reply:
https://emotion.sh/docs/nested